ptr.py

```diff
@@ -40,8 +48,8 @@ def _config_default() -> ConfigParser:
 def _config_read(cwd: str, conf_name: str = ".ptrconfig") -> Optional[ConfigParser]:
     """ Look from cwd to / for a "conf_name" file - If so read it in """
     cp = None
-    cwd_path = Path(cwd)  # type: Optional[Path]
-    root_path = Path("/")
+    cwd_path = Path(cwd)  # type: Path
+    root_path = Path("{}\\".format(cwd_path.drive)) if WINDOWS else Path("/")
```

Comment:
Can you just use `cwd.anchor` here?

Reply:
HOT - TIL - Thanks.

Reply:
It didn't work? I thought line 52 could be replaced entirely by `root_path = cwd.anchor`